packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -132,6 +135,25 @@ class ScrollBehavior {
   /// impossible to select text in scrollable containers and is not recommended.
   Set<PointerDeviceKind> get dragDevices => _kTouchLikeDeviceTypes;
 
+  /// A set of [LogicalKeyboardKey]s that, when any or all are pressed in
+  /// combination with a [PointerDeviceKind.mouse] pointer scroll event, will
+  /// flip the axes of the scroll input.
+  ///
+  /// This will for example, result in the input of a vertical mouse wheel, to
+  /// move the [ScrollPosition] of a [ScrollView] with an [Axis.horizontal]
+  /// scroll direction.
+  ///
+  /// If other keys exclusive of this set are pressed during a scroll event, in
+  /// conjunction with keys from this set, the scroll input will still be
+  /// flipped.
+  ///
+  /// Defaults to [LogicalKeyboardKey.shiftLeft],
+  /// [LogicalKeyboardKey.shiftRight], cannot be null.
```

Comment:
Do you mean this?
```suggestion
  /// [LogicalKeyboardKey.shiftRight], cannot be empty.
```
I mean, it can't be null either, but that's enforced by the compiler.

Reply:
I think it is ok for it to be empty. I guess we don't need to specify it cannot be null, just a habit.

This makes me think when we remove all of the `assert(nonNullableThing != null)`, we should check for doc comments like this too.

Reply:
Looks like we're removing them now. I'm not sure there's an easy way to find all of the places where we document assertions like that, but we should try and find them.

cc @goderbauer